tests/test_rubicon.py

```diff
@@ -260,6 +269,46 @@ def test_mutator_like_method(self):
         # ...at which point it's fair game to be retrieved.
         self.assertEqual(obj1.specialValue, 37)
 
+    def test_ambiguous_fields_as_methods(self):
+        "A method that is shadowed by a property can be accessed as a method."
+        Example = ObjCClass('Example')
+
+        # A class property can be invoked as a method
+        self.assertEqual(Example.classAmbiguous(), 37)
+
+        # An instance property can be invoked as a method
+        obj1 = Example.alloc().init()
+        self.assertEqual(obj1.ambiguous(), 42)
+
+        # In Sierra, mainBundle was turned into a class property.
+        # It should be accessible as a method
+        NSBundle = ObjCClass('NSBundle')
+        try:
+            NSBundle = ObjCClass('NSBundle')
+            NSBundle.mainBundle()
+        except:
```

Comment:
Noo, not a bare `except`, think of the poor tracebacks that get lost there! `:)` I'm not sure, doesn't a test automatically count as failed if it raises an unexpected exception?